### ISSUE TITLE
[Feature Request] Branch per BLE stack?

### ISSUE THREAD

**Issue by LeeNX** [lemmingDev/ESP32-BLE-Gamepad#142]
Maybe more a suggestion that a feature request, possible to keep both ```NimBLE``` and ```Bluedroid``` branches on github? I see that NimBLE is in place, but using zip archives files for the Bluedroid stack in issues, must add unneeded overhead for the developer. Switching between branches locally should be easy, thou I am no git guru.

**Comment by lemmingDev** [comment]
Yeah - was going to do that, but ran out of time/motivation last holidays after making an updated Bluedroid version for a few people who were chasing it

**Comment by LeeNX** [comment]
I was poking at the [T-vK/ESP32-BLE-Keyboard](https://github.com/T-vK/ESP32-BLE-Keyboard) and saw that it supports both libs with ```#if``` defines, like - https://github.com/T-vK/ESP32-BLE-Keyboard/blob/f8dd4852113a722a6b8dc8af987e94cf84d73ad5/BleKeyboard.cpp#L3. 
  
I was thinking possibly three branchs, like main that supports both with the define and two cleaner branchs for each lib. It's a bit more work, but gives us all three options.  
  
See what I can hack up, soon.

**Comment by lemmingDev** [comment]
Interesting

Any help appreciated!

**Comment by LeeNX** [comment]
I think we can close this issue too, @lemmingDev what do you think?

**Comment by lemmingDev** [comment]
Yeah

**Comment by LeeNX** [comment]
Just for reference - https://github.com/espressif/arduino-esp32/discussions/10991#discussioncomment-12259976

**Comment by lemmingDev** [comment]
Thanks for the heads up
